types/events_test.go

```diff
@@ -10,18 +10,18 @@ import (
 func TestQueryTxFor(t *testing.T) {
 	tx := Tx("foo")
 	assert.Equal(t,
-		fmt.Sprintf("tm.event='Tx' AND tx.hash='%X'", tx.Hash()),
+		fmt.Sprintf("tm.event = 'Tx' AND tx.hash = '%X'", tx.Hash()),
 		EventQueryTxFor(tx).String(),
 	)
 }
 
 func TestQueryForEvent(t *testing.T) {
 	assert.Equal(t,
-		"tm.event='NewBlock'",
+		"tm.event = 'NewBlock'",
```

Comment:
does the new code also handle these cases without spaces around the equals?

Reply:
Yes—but the way the tests are written, the expect the output string to be identical to the input string. So this is just a bodge to avoid having to reimplement the round-trip test for every case.